src/renderer/item.store.ts

```diff
@@ -9,7 +9,7 @@ export interface ItemObject {
 
 @autobind()
 export abstract class ItemStore<T extends ItemObject = ItemObject> {
-  abstract loadAll(): Promise<void>;
+  abstract loadAll(...args: any[]): Promise<any>;
```

Comment:
Why do we want `loadAll` to return anything?
```suggestion
  abstract loadAll<T extends any[]>(...args: T): Promise<any>;
```